examples/interconnects.py

```diff
@@ -146,10 +155,10 @@
     print(e.msg['message'])
 
 # Get name servers
-print("Get name servers that matches ID 929f265b-30dc-44bd-97f7-0942d56e9939")
+print("Get name servers that matches ID that matches tha specified ID")
```

Comment:
I believe "Get the named servers for the interconnect that matches the specified ID" sounds better.